### PR TITLE
add endpoint for neha /course/<param>

### PR DESCRIPTION
**Pull request by KevinLi3**
PR containing endpoint, modifying course page to use just uw course schema and not Course equivalency

Ready for merging!

### CODE REVIEW

File: api/app.py [hunk 1]
```diff
@@ -213,7 +219,7 @@ def login_success():
 @app.route('/get_uni/<param>', methods=['GET'])
 def get_uni(param):
     uni = University.query.filter(University.id == param).first()
-    res = uni_schema.dump(uni)
+    res = uwcourse_schema.dump(uni)
```
Comment (aparajitaghimire) [inline api/app.py]: why is it querying for unis but dumping w/ uw_course?

Comment (aparajitaghimire) [reply api/app.py]: it should just b dumping with uni schema

File: api/migrations/versions/c3a8a01b1af0_merging_two_heads.py [hunk 2]
```diff
@@ -0,0 +1,24 @@
+"""merging two heads
+
+Revision ID: c3a8a01b1af0
+Revises: 8dfdbaa32bf6, a3a49ed72b3a
```
Comment (nehageorge) [inline api/migrations/versions/c3a8a01b1af0_merging_two_heads.py]: Remove both migrations

**Review by nehageorge — COMMENTED** [review]
Pls remove migrations and rename endpoint to uwcourse :)

File: api/migrations/versions/c3a8a01b1af0_merging_two_heads.py (hunk 2 above)
Comment (KevinLi3) [reply api/migrations/versions/c3a8a01b1af0_merging_two_heads.py]: Resolved

File: api/app.py (hunk 1 above)
Comment (KevinLi3) [reply api/app.py]: resolved

File: api/migrations/versions/ef52e75a2aec_.py [hunk 3]
```diff
@@ -0,0 +1,38 @@
+"""empty message
```
Comment (nehageorge) [inline api/migrations/versions/ef52e75a2aec_.py]: migration file should have a name after the underscore and that name should be in the message

**Review by nehageorge — COMMENTED** [review]
lgtm mostly

File: api/migrations/versions/ef52e75a2aec_.py (hunk 3 above)
Comment (KevinLi3) [reply api/migrations/versions/ef52e75a2aec_.py]: resolved

File: src/App.js [hunk 4]
```diff
@@ -32,10 +32,12 @@ function App() {
           <Route exact path="/universities" element={<UniversitySearch />} />
           <Route exact path="/get_uni/:id/:page" element={<ViewUni />} />
           <Route exact path="/course/home" element={<CourseHome />} />
+          <Route exact path="/uwcourse/:id" element={<CoursePage />} />
           <Route
             exact
-            path="/get_uni/:id/:page/:course_name"
-            element={<CoursePage />}
+            path="/course/prevSequence"
+            element={<PreviousCourseSequences />}
```
Comment (aparajitaghimire) [inline src/App.js]: I think this shouldn't be added (PreviousCourseSequences)

Comment (KevinLi3) [reply src/App.js]: resolved

**Review by aparajitaghimire — APPROVED** [review]

**Pull request merged**